workflow/rules/trim.smk

```diff
@@ -32,7 +32,7 @@ rule cutadapt_pe:
     log:
         "logs/cutadapt/{sample}-{unit}.log",
     params:
-        others=config["params"]["cutadapt-pe"],
+        extra=config["params"]["cutadapt-pe"],
```

Comment:
Very important fix, to ensure the `params` from the `config.yaml` are actually picked up!